.travis.yml

```diff
@@ -74,14 +71,10 @@ matrix:
     - os: osx
       language: generic
       env:
-        - PYTHON=3.6.6
+        - PYTHON=3.6
         - NUMPY=1.15
         - SCIPY=1.2
         - BUILD_INIT=tools/ci/travis_pip.sh
-    - os: osx
-      language: generic
-      env:
-        - PYTHON=3.7
```

Comment:
And this is the conda/mkl version of python.

Reply:
I restored this one since azure doesn't use conda.